### PR TITLE
JSGEN-47 - fixes

### PR DESCRIPTION
**Pull request by maciejmatu**
- add `printWidth` to keep settings consistent between eslint and prettier
- update incorrect prop-types
- modify svg mock to work with both path and component imports
- keep enzyme config in a single setup file
- fix RoleCard and RoleList storybook stories
- update formatting on sparta page to avoid using `eslint-disable`

### CODE REVIEW

File: __tests__/pages/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -13,11 +13,7 @@ exports[`IndexPage page renders correctly 1`] = `
...
     animationStartValue={0}
     chip={null}
-    image={
-      Object {
-        "ReactComponent": "icon-mock",
-      }
-    }
+    image="icon-mock"
     indent={false}
     title="A user governed video platform"
```
Comment (blrhc) [inline __tests__/pages/__snapshots__/index.test.js.snap]: "user-governed" vs user governed

File: src/pages/sparta/index.js [hunk 2]
```diff
@@ -45,19 +55,10 @@ const SpartaPage = ({ content }) => {
         chip={<Chip onClick={() => setModalOpen(true)}>What is this?</Chip>}
       >
         <p className="SpartaPage__hero-paragraph">
-          Explore available roles and pick the one that suits you the most. Influence platforms development earning
-          Monero in the process.
+          Explore available roles and pick the one that suits you the most.
+          Influence platforms development earning Monero in the process.
```
Comment (blrhc) [inline src/pages/sparta/index.js]: Influence _the platform's_ development earning Monero in the process

**Comment by blrhc** [comment]
### High-level review
The website behaves as expected. I could not find any broken links or errors in the expected functionality. The main problem (unconnected to any changes made in this pull request) was spelling mistakes or incorrect information which has not been updated.

Here are some of the significant cases I identified:
- The footer still contains a link to rocket.chat, which we are no longer using for support purposes.
- The subtitle of the homepage says _Earn Monero by participating in the current Athens testnet_; we are no longer on Athens, have moved onto Acropolis and will soon be launching Rome.
- The homepage section on the Acropolis Testnet is titled `Acropolis Tesnet` (misspelled) followed by a button which says `Explore acropolis` rather than `Explore Acropolis`

Some less severe spelling/grammar issues below:
- On the roles page, the subtitle says `Explore available roles and pick the one that suits you the most`, `the best` would flow much better.
- On the roles page, the subtitle also says `Influence platforms development earning Monero in the process.`, `Influence the platform's` would be correct.
- Both of the above errors are repeated on the `/acropolis` page.
- Under the council member's responsibilities, `Represent the community members and your constituency to make day to day operations decisions` is included. `day to day` ought to be replaced by `day-to-day`
- Under the storage provider's responsibilities, `Run an and maintain storage nodes` is included. `an` should be replaced with `and`

I did also notice that there is no information on Rome (our upcoming testnet) on the main page and there is no `/rome` page on the rebranded website. Has it just not been designed yet?

**Comment by maciejmatu** [comment]
@blrhc thanks for the review! I will update all the spelling errors :+1: Rome testnet info is added in PR #48, this one was just to fix some minor issues in the codebase.

**Comment by maciejmatu** [comment]
Should I remove rocket.chat link, or replace it with a different one?

**Comment by blrhc** [comment]
> @blrhc thanks for the review! I will update all the spelling errors 👍 Rome testnet info is added in PR #48, this one was just to fix some minor issues in the codebase.

Perfect, thanks!

**Comment by maciejmatu** [comment]
@blrhc fixed all the issues you pointed out except for the last one. Not sure what this sentence is supposed to convey. 😄Maybe @bedeho  can help here?

**Comment by blrhc** [comment]
> @blrhc fixed all the issues you pointed out except for the last one. Not sure what this sentence is supposed to convey. 😄Maybe @bedeho can help here?

Other than that bit...

**Comment by blrhc** [comment]
@bwhm Can we merge this?

**Comment by bwhm** [comment]
Sure.

From now on, please make new PRs to development branch.

**Pull request merged**